scripts/updates/api/termux_github_api_get_tag.sh

```diff
@@ -1,95 +1,84 @@
 # shellcheck shell=bash
 termux_github_api_get_tag() {
-	if [[ -z "$1" ]]; then
-		termux_error_exit <<-EndOfUsage
-			Usage: ${FUNCNAME[0]} PKG_SRCURL [TAG_TYPE [FILTER_REGEX]]
-			Returns the latest tag of the given package.
-		EndOfUsage
-	fi
-
 	if [[ -z "${GITHUB_TOKEN:-}" ]]; then
 		# Needed to use graphql API.
 		termux_error_exit "GITHUB_TOKEN environment variable not set."
 	fi
 
-	local PKG_SRCURL="$1"
-	local TAG_TYPE="${2:-}"
-	local FILTER_REGEX="${3:-}"
+	local project tag_type filter_regex=""
 
-	local project
-	project="$(echo "${PKG_SRCURL}" | cut -d'/' -f4-5)"
-	project="${project#git+}"
+	project="$(cut -d'/' -f4-5 <<<"${TERMUX_PKG_SRCURL}")"
```

Comment:
I replaced this with an approach based on the `TERMUX_PKG_SRCURL` portion of `scripts/lint-packages.sh` in `termux_gitlab_api_get_get.sh`
https://github.com/termux/termux-packages/pull/27576/files#diff-9f5afb93772b963d47baa4974f7647f9dc579c91ec9f9ab926d334f2f535e3f0R32-R37
https://github.com/termux/termux-packages/blob/ba62e392be646158b36fed30dddedf2215cfbf31/scripts/lint-packages.sh#L474

The same can probably done here as well.